modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java

```diff
@@ -82,6 +84,13 @@ else if (acceptLanguage.equals("zh-Hant-TW")) {
 				});
 
 			if (ListUtil.isEmpty(locales)) {
+				if (Objects.equals(
+						Http.Method.valueOf(_httpServletRequest.getMethod()),
+						Http.Method.GET)) {
+
+					return Collections.emptyList();
```

Comment:
Does it make sense to perform an extra if clause and return an empty list if this code **is only executed when there is an empty list**?

Reply:
I mean, should not be better to include only one if clause with both conditions?